src/content/en/fundamentals/discovery-and-monetization/payment-request/index.md

```diff
@@ -40,7 +40,8 @@ like to talk about a more comprehensive solution.
 
 ## Introducing the Payment Request API {: #introducing }
 
-The Payment Request API is a system that is meant to *eliminate checkout forms*.
+The Payment Request API is a [W3C standard](https://www.w3.org/TR/payment-request/)
+that is meant to *eliminate checkout forms*.
```

Comment:
I didn't want to call it a standard because it was only a candidate. You may change it to "W3C standard candidate".